odissei/ns/mcal/.htaccess

```
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Options +FollowSymLinks
+RewriteEngine on
+
+RewriteRule ^/ns/mcal/(.*)$ http://api.odissei.triply.cc/redirect/https://w3id.org/odissei/ns/mcal/$1 [R=307,L]
```

RewriteRules are relative to the current directory.  I'm guessing you wanted this:
```suggestion
RewriteRule ^(.*)$ http://api.odissei.triply.cc/redirect/https://w3id.org/odissei/ns/mcal/$1 [R=307,L]
```